aca_preview/preview.py

```diff
@@ -517,6 +518,7 @@ def check_catalog(self):
                 self.check_pos_err_guide(guide_star)
                 self.check_imposters_guide(guide_star)
                 self.check_too_bright_guide(guide_star)
+                self.check_bad_pm(guide_star)
```

Comment:
Call this `check_bad_agasc_proper_motion`.  We can be descriptive with the check names.